### DISCUSSION TITLE
Tracing configuration

### DISCUSSION THREAD

**Discussion by javatechno** [apache/camel-karavan#1217] [Q&A] [answered]
Referring to this discussion: https://github.com/apache/camel-karavan/discussions/714 . It is correct that to enable tracing in created Karavan projects property **camel.main.backlog-tracing = true** must be placed in **/karavan-app/src/main/resources/application.properties** for Karavan web application?

Or this property must be somehow passed to camel-jbang cmd in /devmode Docker container?

Sorry for such questions, I'm not quite familiar with Karavan ecosystem yet.

**Comment by davsclaus** [comment, marked as answer]
you just create an application.properties file in the same location as your camel.yaml file.
